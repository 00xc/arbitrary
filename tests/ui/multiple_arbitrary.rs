use arbitrary::Arbitrary;

#[derive(Arbitrary)]
struct Point {
    #[arbitrary(value = 2)]
    #[arbitrary(value = 3)]
    x: i32,
    y: i32
}

fn main() { }
