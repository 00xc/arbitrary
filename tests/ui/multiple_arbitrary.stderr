error: proc-macro derive panicked
 --> tests/ui/multiple_arbitrary.rs:3:10
  |
3 | #[derive(Arbitrary)]
  |          ^^^^^^^^^
  |
  = help: message: Multiple conflicting #[arbitrary] attributes found on field `x`
